src/app/(sidebar)/account/create/page.tsx

```diff
@@ -46,27 +92,61 @@ export default function CreateAccount() {
             <Button size="md" variant="secondary" onClick={generateKeypair}>
               Generate keypair
             </Button>
-            {IS_TESTING_NETWORK ? (
+
+            {IS_TESTING_NETWORK || IS_CUSTOM_NETWORK_WITH_HORIZON ? (
               <Button
                 size="md"
+                disabled={!account.publicKey || isLoading}
                 variant="tertiary"
-                onClick={() => router.push(Routes.ACCOUNT_FUND)}
+                onClick={() => refetch()}
+                data-testid="fundAccount-button"
               >
                 Fund account with Friendbot
               </Button>
             ) : null}
           </div>
 
-          <ExpandBox isExpanded={Boolean(account.publicKey)} offsetTop="xl">
-            <div className="Account__result">
-              <GenerateKeypair
-                publicKey={account.publicKey}
-                secretKey={secretKey}
-              />
-            </div>
-          </ExpandBox>
+          {Boolean(account.publicKey) && (
+            <ExpandBox isExpanded={Boolean(account.publicKey)} offsetTop="xl">
+              <div className="Account__result">
+                <GenerateKeypair
+                  publicKey={account.publicKey}
+                  secretKey={IS_TESTING_NETWORK ? account.secretKey : secretKey}
+                />
+              </div>
+            </ExpandBox>
+          )}
         </div>
       </Card>
+
+      {showAlert && isFetchedAfterMount && isSuccess && account.publicKey && (
+        <Alert
+          placement="inline"
+          variant="success"
+          actionLabel="View on stellar.expert"
+          actionLink={`https://stellar.expert/explorer/${network.id}/account/${account.publicKey}`}
```

Comment:
Should we hide the StellarExpert link for the `futurenet` and `custom` networks?